shortcircuit-api/src/main/java/com/powsybl/shortcircuit/ShortCircuitConstants.java

```diff
@@ -15,4 +15,6 @@ private ShortCircuitConstants() {
     }
 
     public static final boolean SUBTRANS_STUDY = false;
+
+    public static final boolean WITH_CONTRIBUTIONS = true;
```

Comment:
Maybe better WITH_FEEDER_CONTRIBUTIONS.

Reply:
are we sure contribution is the right word for that? Looks like badly french translated :-)